.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git/*
+test/*
+*/*_test.go
```

Comment:
Since you're using multistage Dockerfile builds, I would not exclude test files. These may be inadvertently ignored by our CI (now or in the future). Ignoring .git and _output makes sense.